inference-engine/src/gna_plugin/gna_model_serial.cpp

```diff
@@ -256,15 +256,21 @@ void GNAModelSerial::Export(void * basePointer, size_t gnaGraphSize, std::ostrea
     header.gnaMemSize = gnaGraphSize;
     header.layersCount = layers.size();
     header.nGroup = guessGrouping(*gna2Model);
-    header.input = convert_to_serial(input);
-    header.output = convert_to_serial(output);
-
+    header.nInputs = inputs.size();
```

Comment:
dont see deserialisation code for vector - putting it inside aligned POD object a mistake